e2e/utils/_.bash

```diff
@@ -268,6 +283,14 @@ get_replica_port() {
   cat "$E2E_NETWORK_DATA_DIRECTORY/replica-configuration/replica-1.port"
 }
 
+get_pocketic_pid() {
+  cat "$E2E_NETWORK_DATA_DIRECTORY/pocket-ic-pid"
+}
+
+get_pocketic_port() {
+  cat "$E2E_NETWORK_DATA_DIRECTORY/pocket-ic.port"
```

Comment:
This can be `pocket-ic-port`, which will be consistent with `pocket-ic-pid` as well as `webserver-port`.  The `replica-1.port` filename is the only one that's inconsistent with everything else, and that is a historical artifact.